docs/src/main/asciidoc/_elytron/OpenSSL.adoc

```diff
@@ -38,4 +38,19 @@ If OpenSSL was loaded successfully, a message similar to the one below will occu
 [source,options="nowrap"]
 ----
 15:37:59,814 INFO [org.wildfly.openssl.SSL] (MSC service thread 1-7) WFOPENSSL0002 OpenSSL Version OpenSSL 1.1.1d FIPS  10 Sep 2019
-----
\ No newline at end of file
+----
+
+=== TLSv1.3
+
+TLSv1.3 will be enabled by default when using OpenSSL 1.1.1 or higher and when running against JDK 11 or higher.
+OpenSSL will use the following TLSv1.3 cipher suites by default:
+
+[source,options="nowrap"]
+----
+TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256
```

Comment:
Just curious - I guess this will need some change into wildfly/wildfly-core code/configuration, right?

Reply:
No additional changes will be needed for this. Support for the `cipher-suite-names` attribute for SSL contexts was already added as part of WFCORE-4172.

Reply:
+1 you are right, of course